bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/database/migration/Migration2to3.kt

```diff
@@ -0,0 +1,11 @@
+package io.bucketeer.sdk.android.internal.database.migration
+
+import androidx.sqlite.db.SupportSQLiteDatabase
+import io.bucketeer.sdk.android.internal.evaluation.db.EvaluationEntity
+
+class Migration2to3 : Migration {
+
+  override fun migrate(db: SupportSQLiteDatabase) {
+    db.delete(EvaluationEntity.TABLE_NAME, null, null)
```

Comment:
Should we recreate `Evaluation` table again after delete ? @cre8ivejp

Reply:
There is no need to recreate the table. I think the approach here is to delete the rows in the table or drop the table and create it again.
The delete operation would be fine since there is only one row in the table.

Also, I wrote a test to check if it is empty after migration.
https://github.com/bucketeer-io/android-client-sdk/pull/79/files#diff-9e9eebfdac59b75de3adc119068791edf454ffff72a7cddfd427a804bbadb7a7R86-R96

Reply:
thank you, I checked it document.
> `delete` Convenience method for deleting rows in the database